vault/audit.go

```diff
@@ -411,6 +411,7 @@ func (c *Core) newAuditBackend(entry *MountEntry, view logical.Storage, conf map
 
 		if c.logger.IsDebug() {
 			c.logger.Debug("audit: adding reload function", "path", entry.Path)
+			c.logger.Debug("audit: file backend options", "path", entry.Path, "file_path", entry.Options["file_path"])
```

Comment:
Although the options map *shouldn't* be empty, if it is it will cause a panic. You should modify the IsDebug check (or in this block, add another check) to also verify Options isn't nil before indexing into it.

Reply:
Thanks for the tip; for clarification, is this the only place it the check on Options not being nil is needed, or same for all three backends?

Reply:
Same for all three please!